conf.py

```diff
@@ -1,5 +1,9 @@
 import sys
 import os
+import requests
+from urllib.parse import urlparse
+from git import Repo
```

Comment:
This can cause a build failure if `gitpython` isn't installed.
To make sure that it is installed, add `gitpython` to the requirements list here: https://github.com/canonical/sphinx-docs-starter-pack/blob/d42ac81f5416987cbf4572c34933cadb75b85225/.sphinx/build_requirements.py#L85

(That is really confusing, I know. I wouldn't expect you to be able to figure this out. :wink: )